libcontainer/cgroups/systemd/v2.go

```diff
@@ -288,6 +290,36 @@ func (m *unifiedManager) Apply(pid int) error {
 	if err := fs2.CreateCgroupPath(m.path, m.cgroups); err != nil {
 		return err
 	}
+
+	// The kernel exposes a list of files that should chowned to
```

Comment:
Perhaps move the code to compute `filesToChown` inside the `if c.OwnerUID != nil {` scope?

Reply:
I shall implement this change in the next push.